### ISSUE TITLE
"New service:" log messages

### ISSUE THREAD

**Issue by glennschmidt** [kubernetes/dns#142]
For the past several days, my cluster config has been stable (no pods being created/deleted, no nodes going up/down). Yet kube-dns is logging hundreds of messages every hour about "new services".

For example:
```
I0818 05:08:18.445594       1 dns.go:264] New service: elasticsearch-logging
I0818 05:08:18.446497       1 dns.go:264] New service: kube-dns
I0818 05:08:18.446159       1 dns.go:264] New service: kubernetes
I0818 05:08:18.446427       1 dns.go:264] New service: heapster
```
It seems to be looping through all the services that exist in the cluster and reporting them as 'new' every 2-3 minutes.

My questions are

1. Is this normal, and
2. If not, does anyone know what could be causing this behavior?

**Comment by bowei** [comment]
What is the version of your kube-dns? It used to log these messages whenever it syncs with the kube-api server, but the verbosity has been raised above v2 so in normal operation these logs should not be present:
```
		glog.V(3).Infof("New service: %v", service.Name)
```

**Comment by glennschmidt** [comment]
Sorry i left that information out
kubedns version is 1.14.1
kubernetes version is 1.6.6

**Comment by bowei** [comment]
What is your vlog level (-v command line arg)

**Comment by glennschmidt** [comment]
Looks like `--v=2` (i haven't set this so it must be the default when deployed with kops)

**Comment by bowei** [comment]
This has been fixed in kube-dns 14.4.
